app/delivery/routes.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const getMeteors = require("../use_cases/meteors_service");
+const router = express.Router();
+
+router.get('/meteors', async (req, res) => {
+    try {
+        const default_date = "2000-01-01";
+        const start_date = req.query.start ?? default_date;
+        const end_date = req.query.end ?? start_date;
```

Comment:
"" => ''
NASA api sets default value for end_date = start_date + 7 days, so if end date is not presented in request, you can skip this param (but your logic looks pretty good)
Also default_date value doesn't look obvious, you could use some kind of `yestarday` or `7 days ago`. 24 years are too much :)

Reply:
Updated this